### ISSUE TITLE
Inconsistent parameter types with respect to other AWS orbs

### ISSUE THREAD

**Issue by ExternalReality** [CircleCI-Public/aws-ecs-orb#94]
AWS orbs that authenticate with secrets such as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` typically use the type `env_var_name` for commands and jobs accepting those parameters. This orb uses the string type. If a user is importing both the `ecr` orb and this orb together (which is quite typical) the inconsistency can be confusing and lead to frustrating configuration errors. Once a user discovers the source of error, they are likely to loose confidence in first party orb offerings.

In summary the `ecr` orb, `s3` orb, and the `aws-cli` orb all use the type `env_var_name` for the authentication parameters of jobs and commands. This orb, `ecs`, uses strings.

The following issues is related and an example of the confusion that can be caused: https://github.com/CircleCI-Public/aws-ecs-orb/issues/9

**Comment by gmemstr** [comment]
Likely to be done when we eventually refactor this orb using the new Orb Development Kit standards.

**Comment by KyleTryon** [comment]
Working on making this a part of #118

**Comment by icco** [comment]
@KyleTryon this looks done?

**Comment by KyleTryon** [comment]
> @KyleTryon this looks done?

Correct 👍 thank you for pointing that out. Closing this issue.
